tests/std/tests/GH_004109_iter_value_t_direct_initialization/env.lst

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+RUNALL_INCLUDE ..\strict_concepts_20_matrix.lst

```

Comment:
This PR changes pre-C++20 algorithms, should use `usual_matrix.lst`? (which requries more conditional compilation)

Reply:
I chose C++20 because that example used:
```
static_assert(random_access_iterator<I>);
static_assert(sortable<I>);
```
Yes, we can use `usual_matrix.lst` if needed